Example/CotterIOS/ViewController.swift

```diff
@@ -56,6 +58,7 @@ class ViewController: UIViewController {
         
         func cbFunc(accessToken: String, error: Error?) -> Void{
             guard let error = error else {
+                self.navigationController?.popToViewController(self, animated: false)
```

Comment:
don't do this here. logically speaking i think this code should live in Cotter. i think it doesn't make sense if client is the one to tell Cotter to pop all VCs in the navigation stack back to itself, instead Cotter itself should be the one to "consciously" pop the navigation stack back to the parent VC, meaning to the VC calling `Cotter.init()` (in this case: this VC). argue me on this if you think otherwise.

example of doing it in Cotter:
https://github.com/cotterapp/ios-sdk/blob/931c8c82b3c4ba5ac4ce6041ed3f8dbf36ce38e6/Source/PINViewController.swift#L233